include/mega/filesystem.h

```diff
@@ -248,7 +252,7 @@ struct MEGA_API FileSystemAccess : public EventTrigger
 
     // convert MEGA-formatted filename (UTF-8) to local filesystem name; escape
     // forbidden characters using urlencode
-    void local2name(string*) const;
+    virtual void local2name(string*) const;
```

Comment:
similarly, this one shouldn't become virtual, it calls the virtual implementation path2local/local2path, the customization should be on those two.

Reply:
Indeed. Will fix.